roqoqo-iqm/src/backend.rs

```diff
@@ -466,52 +410,65 @@ impl Backend {
             eprintln!("Warnings: {:?}", iqm_result.clone().warnings.unwrap());
         }
 
-        if iqm_result.status == Status::Failed {
-            return Err(RoqoqoBackendError::GenericError {
-                msg: format!(
-                    "Job FAILED with job ID: {}\nMessage: {}",
-                    id,
-                    iqm_result.message.unwrap()
-                ),
-            });
-        }
-
```

Comment:
How come this has been removed?

Reply:
Because all the status checks have been moved to the calling function `wait_for_results`